mapie/tests/test_classification.py

```diff
@@ -530,9 +530,6 @@ def predict_proba(self, X: NDArray, *args: Any) -> NDArray:
     def predict(self, X: NDArray, *args: Any) -> NDArray:
         return np.repeat(1, len(X))
 
-    def _get_param_names(self):
```

Comment:
Why removing this method ? No longer used ?

Reply:
It does not seem to be needed and the coverage is not at 100% due to this. Not sure of the additional value of this mehod.